lib/chef_zero/endpoints/cookbook_artifacts_endpoint.rb

```diff
@@ -9,10 +9,10 @@ def get(request)
 
         artifacts = begin
           list_data(request)
```

Comment:
was this autocorrected? feels odd that this line wasn't indented past begin as well. but super nit-pick.

Reply:
That wasn't. I can see if it complains when I do it. It is hard to read like this.

Reply:
yeah i think the reason why this happens is mildly complicated, but i've just been manually fixing them.

i usually throw the begin on a new line and that keeps it all more left justified when it gets autofixed by my editor.

vim finally does support the begin being on the same line and lining up the end statement only indented over twice from the start of the line, which is really how i'd prefer things in an ideal world (and opening brackets on their own line in C code is an abomination).